### ISSUE TITLE
Use require for poly-decomp

### ISSUE THREAD

**Issue by liabru** [liabru/matter-js#160]
Use require for `poly-decomp`. But do not include `poly-decomp` in the build for Matter.js. Ensure that this also works in a browser, where `poly-decomp` is required separately.

**Comment by donaldpipowitch** [comment]
I use it with webpack in this way for now:

````
import decomp from 'poly-decomp';
import { Engine, World, Bodies, Svg, Vertices } from 'matter-js';

window.decomp = decomp;
````

(By the way: Using webpack gives a warning `This seems to be a pre-built javascript file. (build/matter.js 32:479-486)`. I don't know if this can make problems in Browserify/webpack environments, because there seems to be a `require` fallback.)

**Comment by liabru** [comment]
Solved as of c72a9d0.
